readme.txt

```diff
@@ -92,6 +92,8 @@ Here are some frequently asked questions about how to use the plugin.
 
 [youtube https://www.youtube.com/watch?v=kYdP0dtueEE]
 
+Note: With the new WordPress editor, which is used in 5.0 and above, you can use the Shortcode block and paste the shortcode "[list_board_members]" into the block to display your board members.
```

Comment:
Do you think we should change "5.0" to be "WordPress 5.0" just to be sure people understand what 5.0 is referring to?

Reply:
Can do.

Reply:
This has been updated.